airflow/config_templates/default_test.cfg

```diff
@@ -102,6 +102,18 @@ default_queue = default
 sync_parallelism = 0
 worker_precheck = False
 
+[mesos]
+master =
+framework_name =
+task_cpu =
+task_memory =
+checkpoint =
+authenticate =
+docker_image_slave =
+docker_volume_dag_driver =
+docker_volume_dag_name =
+docker_volume_dag_container_path =
+
```

Comment:
We don't need to add things to test.cfg unless we need explicit default values for tests to run. Do we need these values to be empty?

Reply:
no, I misunderstood it. I'm sorry.